internal/utils/dataplane/config.go

```diff
@@ -75,3 +84,26 @@ func FillDataPlaneProxyContainerEnvs(existing []corev1.EnvVar, podTemplateSpec *
 	}
 	sort.Sort(k8sutils.SortableEnvVars(container.Env))
 }
+
+// ConfigureKongPluginRelatedEnvVars returns the environment variables
+// needed for configuring the Kong Gateway with the provided Kong Plugin
+// names. If kongPluginNames is nil or empty, nil is returned.
```

Comment:
```suggestion
// names. If kongPluginNames is nil or empty, nil is returned. Kong will use bundled
// plugins by default if we do not override `KONG_PLUGINS`.
```
Just to make it clear we do not make a mistake here by returning nil.